package.json

```diff
@@ -34,7 +34,7 @@
   },
   "peerDependencies": {
     "@aws-sdk/credential-providers": "^3.201.0",
-    "mongodb-client-encryption": "^2.3.0",
+    "mongodb-client-encryption": ">=2.3.0",
```

Comment:
I think we can actually consider an official change to the semver range we use here to make betas easier to ingest: `>=2.3.0 <3`, this will prevent the 5.0 driver from being automatically compatible with the future 3.x major. (cc @dariakp)

Reply:
Updated to `>=2.3.0 <3`

Reply:
makes sense to me